$OpenBSD: patch-configure,v 1.3 2019/05/05 10:24:26 jca Exp $

CR_ERROR_FIRST isn't shipped by recent versions of mariadb and isn't
used anywhere in the dspam source code.

Index: configure
--- configure.orig
+++ configure
@@ -13280,7 +13280,7 @@ then
     #include <mysql.h>
     #include <mysqld_error.h>
     #include <errmsg.h>
-    #if defined(PROTOCOL_VERSION) && defined(ER_LOCK_DEADLOCK) && defined(ER_LOCK_WAIT_TIMEOUT) && defined(ER_LOCK_OR_ACTIVE_TRANSACTION) && defined(CR_ERROR_FIRST)
+    #if defined(PROTOCOL_VERSION) && defined(ER_LOCK_DEADLOCK) && defined(ER_LOCK_WAIT_TIMEOUT) && defined(ER_LOCK_OR_ACTIVE_TRANSACTION)
     /* Success */
     #else
     #error Unsupported version of MySQL
